From e20dfda5b6f85b67e82d19bc63acfd5611ca98c6 Mon Sep 17 00:00:00 2001
From: Tim S <tim.stahlhut@gmail.com>
Date: Sun, 27 Jan 2019 13:33:54 -0500
Subject: Edit config.sub

This patch is based on the work of 
Brian Dominy https://github.com/bcd/newlib-6809 and
David Flamand https://gitlab.com/dfffffff/newlib-6809

---
 config.sub | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/config.sub b/config.sub
index 62d9cbe..b1a2d18 100644
--- a/config.sub
+++ b/config.sub
@@ -289,7 +289,7 @@ case $basic_machine in
 	| z8k)
 		basic_machine=$basic_machine-unknown
 		;;
-	m6811 | m68hc11 | m6812 | m68hc12)
+	m6809 | m6811 | m68hc11 | m6812 | m68hc12)
 		# Motorola 68HC11/12.
 		basic_machine=$basic_machine-unknown
 		os=-none
@@ -330,6 +330,7 @@ case $basic_machine in
 	| i*86-* | i860-* | i960-* | ia64-* \
 	| ip2k-* | iq2000-* \
 	| m32c-* | m32r-* | m32rle-* \
+	| m6809-* \
 	| m68000-* | m680[012346]0-* | m68360-* | m683?2-* | m68k-* \
 	| m88110-* | m88k-* | maxq-* | mcore-* \
 	| mips-* | mipsbe-* | mipseb-* | mipsel-* | mipsle-* \
@@ -459,6 +460,10 @@ case $basic_machine in
 		basic_machine=c90-cray
 		os=-unicos
 		;;
+	coco)
+		basic_machine=coco
+		os=-none
+		;;
 	convex-c1)
 		basic_machine=c1-convex
 		os=-bsd
-- 
2.19.1.windows.1

